forc/src/ops/forc_init.rs

```diff
@@ -106,82 +74,92 @@ pub fn init(command: InitCommand) -> Result<()> {
-            defaults::default_manifest(&neat_name, constants::LIB_ENTRY),
+            Path::new(&project_dir).join(constants::MANIFEST_FILE_NAME),
+            defaults::default_manifest(&project_name, constants::LIB_ENTRY),
         )?,
         _ => fs::write(
-            Path::new(&project_name).join(constants::MANIFEST_FILE_NAME),
-            defaults::default_manifest(&neat_name, constants::MAIN_ENTRY),
+            Path::new(&project_dir).join(constants::MANIFEST_FILE_NAME),
+            defaults::default_manifest(&project_name, constants::MAIN_ENTRY),
         )?,
     }
 
     // Insert default test manifest file
     fs::write(
-        Path::new(&project_name).join(constants::TEST_MANIFEST_FILE_NAME),
-        defaults::default_tests_manifest(&neat_name),
+        Path::new(&project_dir).join(constants::TEST_MANIFEST_FILE_NAME),
+        defaults::default_tests_manifest(&project_name),
     )?;
 
-    // Insert src based on program_type
     match program_type {
         Contract => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::MAIN_ENTRY),
             defaults::default_contract(),
         )?,
         Script => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::MAIN_ENTRY),
             defaults::default_script(),
         )?,
         Library => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::LIB_ENTRY),
             defaults::default_library(&project_name),
         )?,
         Predicate => fs::write(
-            Path::new(&project_name)
+            Path::new(&project_dir)
                 .join("src")
                 .join(constants::MAIN_ENTRY),
             defaults::default_predicate(),
         )?,
     }
 
     // Insert default test function
-    fs::write(
-        Path::new(&project_name).join("tests").join("harness.rs"),
-        defaults::default_test_program(&project_name),
-    )?;
+    let harness_path = Path::new(&project_dir).join("tests").join("harness.rs");
+    fs::write(&harness_path, defaults::default_test_program(&project_name))?;
+
+    if command.verbose {
+        println_light_blue(&format!(
+            "\nCreating test harness at {}",
+            canonicalize(&harness_path)?
+        ));
+    }
 
     // Ignore default `out` and `target` directories created by forc and cargo.
-    fs::write(
-        Path::new(&project_name).join(".gitignore"),
-        defaults::default_gitignore(),
-    )?;
+    let gitignore_path = Path::new(&project_dir).join(".gitignore");
+    fs::write(&gitignore_path, defaults::default_gitignore())?;
+
+    if command.verbose {
+        println_light_blue(&format!(
+            "\nCreating .gitignore at {}",
+            canonicalize(&gitignore_path)?
+        ));
+    }
```

Comment:
Maybe we should move this earlier to before we write to `gitignore_path` above?

Reply:
Same as comment above regarding past tense